main.js

```diff
@@ -209,13 +209,23 @@ async function initialize_server(con) {
 
       router.post('/fetch_feed', function(req,res,next) {
         if (req.session && req.session.email) {
-          con.query("SELECT * FROM posts LEFT JOIN users ON posts.userid=users.uid ORDER BY posts.id DESC LIMIT 10", [], function (err, result) {
+          // con.query("SELECT * FROM posts LEFT JOIN users ON posts.userid=users.uid ORDER BY posts.id DESC LIMIT 10", [], function (err, result) {

```

Comment:
Move/remove commented out code (Code Style Guide 8`
```suggestion
```

Reply:
I commented this line because I want to keep it if we face a bug later , so that I remember what was the original query

Reply:
Old code may be located in the repo history (this is one of the reasons version control exists).

Reply:
What do you mean by repo history

Reply:
Git stores every commit we've ever made. If you want to revert to this commented out code it may be found in one of these commits.